ccm_web/client/src/pages/BulkSectionCreate.tsx

```diff
@@ -0,0 +1,534 @@
+import { Backdrop, Box, Button, CircularProgress, Grid, Link, makeStyles, Paper, Typography } from '@material-ui/core'
```

Comment:
Optional: Will it make sense to use `features` instead of `pages`? since it mostly all the features starting landing page goes in here. In MyLA we are more using Container convention.

Reply:
I like `pages`, there might be other use cases for container-like components that isn't a feature, and I prefer `page` over `container`.

Reply:
I think if you look at the naming convention used, some of the file names says `Featureard` and `FeatureUiData`. So I was looking where is the actual features landing file. I found that stuff under `pages` and it was not fit with naming convention followed rest of the files. Just an opinion :)

Reply:
We could create a subdirectory inside `pages` called `features`. I dunno, it's all debatable I guess.

Reply:
I don't really have an opinion.  Except that they're all components.